.changeset/calm-rats-matter.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": major
+---
+
+Delete "closed" from MenuClassKey
```

Comment:
Why remove the `closed` class-key?

Reply:
@juliawegmayr could you please provide an answer here?